### PR TITLE
Minor updates to `metro`

### PR DESCRIPTION
**Pull request by yalinli2**
Minor updates to:
1. remove some redundant codes of `PrimaryClarifierBSM2`.
2. add a warning for `get_SRT` when wasted biomass is calculated to be 0.
3. bring in recent changes from `main`

Thanks!

### CODE REVIEW

File: qsdsan/sanunits/_clarifier.py [hunk 1]
```diff
@@ -979,64 +978,35 @@ class PrimaryClarifierBSM2(SanUnit):
...
-        else:
-            raise ValueError('correction factor expected from user')
    
     def _f_i(self):
         xcod = self._mixed.composite('COD', particle_size='x')
         fx = xcod/self._mixed.COD
-        corr = self._corr
-        HRT = self._HRT
-        n_COD = corr*(2.88*fx - 0.118)*(1.45 + 6.15*np.log(HRT*24*60))
+        n_COD = self.f_corr*(2.88*fx - 0.118)*(1.45 + 6.15*np.log(self.HRT*24*60))
```
Comment (joyxyz1994) [inline qsdsan/sanunits/_clarifier.py:1009]: Looks like `f_corr` is not defined here

**Comment by joyxyz1994** [comment]
Everything else looks good to me! Thanks, @yalinli2!

File: qsdsan/sanunits/_clarifier.py (hunk 1 above)
Comment (joyxyz1994) [reply qsdsan/sanunits/_clarifier.py:1009]: Nvm!

**Comment by yalinli2** [comment]
@RaiSaumitra I'll merge this in since I don't really expect it to affect your codes, let me know if you do notice changes in results, thanks!

**Pull request merged**